bin/ch/WScriptJsrt.cpp

```diff
@@ -490,9 +495,39 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             LPCWSTR errorMessage = nullptr;
             size_t errorMessageLength = 0;
 
-            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"message", &messagePropertyId), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, messagePropertyId, &messageProperty), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength), false);
+            LPCWSTR errorName = nullptr;
+            size_t errorNameLength = 0;
+
+            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"name", &namePropertyId), false);
```

Comment:
> situation due to expectation that stack is always available.

Yes. In my suggestion, I am using your code to extract `stack` only if it is available or print one liner `name:message`. 

> I'm not sure if this very last plug of the binary should be used for testing the framework itself. 

Sorry, I didn't follow this. Could you please elaborate?
>I'm also in favor of printing warning message if any one of IfJsrtErrorFail fails inside this method.

That's a good point. Feel free to update this PR or submit another PR for it.